lib/IniProcessor/ini_processing.cpp

```diff
@@ -653,6 +653,43 @@ bool IniProcessing::hasKey(const std::string &keyName) const
     return (e != m_params.currentGroup->end());
 }
 
+bool IniProcessing::renameKey(const std::string &oldName, const std::string &newName)
+{
+    if(!m_params.opened)
+        return false;
+
+    if(!m_params.currentGroup)
+        return false;
+
+    auto it = m_params.currentGroup->find(oldName);
+    if(it != m_params.currentGroup->end())
+    {
+        std::swap((*m_params.currentGroup)[newName], it->second);
+        m_params.currentGroup->erase(it);
+        return true;
+    }
+
+    return false;
+}
+
+bool IniProcessing::deleteKey(const std::string &keyName)
+{
+    if(!m_params.opened)
+        return false;
+
+    if(!m_params.currentGroup)
+        return false;
+
+    auto it = m_params.currentGroup->find(keyName);
+    if(it != m_params.currentGroup->end())
+    {
+        m_params.currentGroup->erase(it);
+        return true;
+    }
+
+    return false;
+}
+
```

Comment:
These things would go into the main repository of IniProcessing too. https://github.com/WohlSoft/IniProcessing

Reply:
Should I commit these changes directly to that repo or submit a PR?

Reply:
I'll take care by myself :)
Just a note that this change will be also copied into main repo of module.

Reply:
Thanks for the notes; for case sensitivity, I wondered about that, but `hasKey` doesn't have it either. Should we add it there too?

Reply:
Ye, it should bein added too. That's a piece that I mised up by myself :eyes: